src/Http/Http.Extensions/src/RequestDelegateFactory.cs

```diff
@@ -229,13 +229,9 @@ private static Expression CreateArgument(ParameterInfo parameter, FactoryContext
             {
                 return BindParameterFromRouteValueOrQueryString(parameter, parameter.Name, factoryContext);
             }
-            else if (parameter.ParameterType.IsInterface)
```

Comment:
Can we update a test to show interfaces are [inferred to be json bodies now](https://github.com/dotnet/aspnetcore/blob/main/src/Mvc/Mvc.ApiExplorer/test/EndpointMetadataApiDescriptionProviderTest.cs#L126)?

Reply:
Yes, will do. Though I think it'll explode by default.

Reply:
Added a test that uses a custom JSON converter to de-serialize an interface